stdlib/database/h2/src/test/java/org/ballerinalang/stdlib/database/sql/SQLActionsTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We don't need to change the year of this file.

Reply:
Okay. Done.